### PR TITLE
[Fix] CON-122 - advanced search modal match mockup 

### PR DESCRIPTION
**Pull request by 404passionFruit**
**JIRA issue**
[CON-122 (comment after testing)](https://oat-sa.atlassian.net/browse/CON-122?focusedCommentId=114171)

**Related PR**
https://github.com/oat-sa/tao-core-ui-fe/pull/242

**Fixies**
- [x] Change version in package.json `"@oat-sa/tao-core-ui": "1.21.1"`
- [x] Change version in package-lock.json
- [x] `cd ./views/build && npm run build`

Currently is in draft, finish updating after releasing and publishing @oat-sa/tao-core-ui v1.20.5

### CODE REVIEW

File: views/package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
         "@oat-sa/tao-core-libs": "0.4.2",
         "@oat-sa/tao-core-sdk": "1.7.0",
         "@oat-sa/tao-core-shared-libs": "1.0.3",
-        "@oat-sa/tao-core-ui": "1.20.4",
+        "@oat-sa/tao-core-ui": "1.20.5",
```
Comment (mike-ionut-mihai-sandu-tao) [inline views/package.json:16]: when you change a piece of code inside any extension you need to update the version also , 
so please go to manifest and increase the version :) according to the semver,
also i see this is done a while ago (x days ) so please contact your peers to review it

**Review by mike-ionut-mihai-sandu-tao — CHANGES_REQUESTED** [review]

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by habemuscode — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Comment by 404passionFruit** [comment]
As Hanna suggested to find existing logic for keycodes in codebase, moving this back to Draft.

**Pull request merged**